pyproject.toml

```
@@ -13,7 +13,7 @@ repository = "https://github.com/honeycombio/beeline-python"
 
 [tool.poetry.dependencies]
 python = ">=3.5, <4"
-libhoney = ">=1.7.0"
+libhoney = "2.1.1"
```

We generally allow a version range for dependencies. I think limiting it to within a major version is good though.

```suggestion
libhoney = "^2.1.1"
```